### PR TITLE
CS-Fixer: Enable `nullable_type_declaration_for_default_null_value`

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: src/Client/GroupsClient.php [hunk 1]
```diff
@@ -63,7 +63,7 @@ public function buildApiUrl()
         return Curl::API_BASE_URL.'/'.Curl::API_VERSION.'/groups/'.$this->group->getKey().'/'.$this->action.'.json?token='.$this->group->getApplication()->getToken();
     }
 
-    public function buildCurlPostFields(Recipient $recipient = null): array
+    public function buildCurlPostFields(?Recipient $recipient = null): array
     {
         $curlPostFields = [
             'token' => $this->group->getApplication()->getToken(),
```
Comment (OskarStark) [inline src/Client/GroupsClient.php:69]: just spotted, that in line 78, $recipient can be null, right?

Comment (slunak) [reply src/Client/GroupsClient.php:69]: yes, can be `null`

**Review by slunak — APPROVED** [review]

**Comment by slunak** [comment]
This is the last PR for cs fixer? I wonder, is it possible to add a check to github actions to detect that cs fixer was not run and make sure it is run? Is it a good practice? I may be wrong, but depending on PHP version changes cs fixer may apply different changes?

**Comment by OskarStark** [comment]
I will create one afterwards

**Comment by OskarStark** [comment]
You can merge this first please

**Comment by slunak** [comment]
Sure, thanks!

**Pull request merged**